.circleci/config.yml

```diff
@@ -451,30 +466,47 @@ jobs:
       - full-checkout
       - attach_workspace:
           at: .
+      - install-ghr
       - run:
           name: Release Carthage archive on GitHub
           command: |
-            GHR=ghr_v0.12.0_darwin_amd64
-            GHR_SHA256=c868ef9fc5dd8c8a397b74d84051d83693c42dd59041cb17b66f90f563477249
-            curl -sfSL --retry 5 --retry-delay 10 -O "https://github.com/tcnksm/ghr/releases/download/v0.12.0/${GHR}.zip"
-            echo "${GHR_SHA256} *${GHR}.zip" | shasum -a 256 -c -
-            unzip "${GHR}.zip"
-            ./${GHR}/ghr -replace "${CIRCLE_TAG}" MozillaAppServices.framework.zip
+            ghr -replace "${CIRCLE_TAG}" MozillaAppServices.framework.zip
   XCFramework release:
     executor: macos
     steps:
       - full-checkout
       - attach_workspace:
           at: .
+      - install-ghr
       - run:
           name: Release XCFramework archive on GitHub
           command: |
-            GHR=ghr_v0.12.0_darwin_amd64
-            GHR_SHA256=c868ef9fc5dd8c8a397b74d84051d83693c42dd59041cb17b66f90f563477249
-            curl -sfSL --retry 5 --retry-delay 10 -O "https://github.com/tcnksm/ghr/releases/download/v0.12.0/${GHR}.zip"
-            echo "${GHR_SHA256} *${GHR}.zip" | shasum -a 256 -c -
-            unzip "${GHR}.zip"
-            ./${GHR}/ghr -replace "${CIRCLE_TAG}" megazords/ios-rust/MozillaRustComponents.xcframework.zip
+            ghr -replace "${CIRCLE_TAG}" megazords/ios-rust/MozillaRustComponents.xcframework.zip
+  build-nimbus-fml:
+    executor: macos
+    steps:
+      - full-checkout
```

Comment:
Definitely for a follow up... Probably not be worth it unless the fml takes a long time to compile, which it doesn't now - 5 mins is not bad at all:

We might be able to cut down some CI time by using `sccache`:
```yaml
- restore_cache:
    name: Restore sccache cache
    key: fml-sccache-cache-macos-{{ arch }}
- full-checkout
- install rust
- setup-rust-target-version
- setup-sccache

.. rest of commands
- save_cache:
    name: Save sccache cache
    key: fml-sccache-cache-macos-{{ arch }}-{{ epoch }}
    paths:
      - "~/Library/Caches/Mozilla.sccache"
- sccache-stats-and-logs
```

Reply:
Ok, I think I'll leave this for a followup.